applications/luci-app-tor-hs/root/usr/share/rpcd/acl.d/luci-app-tor-hs.json

```diff
@@ -0,0 +1,16 @@
+{
+	"luci-app-tor-hs": {
+		"description": "Grant access to Tor hidden service configurator",
+		"read": {
+			"ubus": {
+				"file":["read","list"],
+				"tor_rpcd.sh": ["list-hs"]
```

Comment:
I see this rpcd is really named **tor_rpcd.sh** https://github.com/openwrt/packages/blob/master/net/tor-hs/Makefile#L61
but it looks wrong to me. I would suggest that we change that in the package feed. 
@ja-pa Is that really how it's meant to be?

Reply:
Renamed to tor-hs-rpc (https://github.com/openwrt/packages/pull/15097)